libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -144,8 +145,24 @@ func (m *Manager) Apply(pid int) (err error) {
 		}
 		m.Paths[sys.Name()] = p
 
+		isPermissionError := func(err error) bool {
```

Comment:
Please define this as an actual function (preferably called `isIgnorableError`) rather than using the `a := func(...) {...}` form.

Reply:
Done. PTAL.